pkg/console/vip.go

```diff
@@ -25,8 +25,12 @@ func createMacvlan(name string) (netlink.Link, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to fetch %s", name)
 	}
-	rand.Seed(time.Now().UnixNano())
-	macvlanName := tempMacvlanPrefix + strconv.Itoa(rand.Intn(100))
+	randNum, err := rand.Int(rand.Reader, big.NewInt(100))
```

Comment:
@yaocw2020 Can you help review this part? Thanks!

Reply:
What are the benefits to use `crypto/rand` instead of `math/rand`?

Reply:
Hi @yaocw2020, thanks for the review.
Using `crypto/rand` instead `math/rand` due to the `G404: Use of weak random number generator (math/rand instead of crypto/rand)` (gosec) checking.